modules/installation-mirror-repository.adoc

```diff
@@ -89,19 +97,29 @@ For `<path_to_pull_secret>`, specify the absolute path to and file name of the p
 
 .. Export the release mirror:
 +
+ifndef::openshift-origin[]
 [source,terminal]
 ----
 $ RELEASE_NAME="ocp-release"
 ----
 +
 For a production release, you must specify `ocp-release`.
+endif::[]
+ifdef::openshift-origin[]
+[source,terminal]
+----
+$ RELEASE_NAME="okd"
+----
+endif::[]
 
-.. Export the type of architecture for your server, such as `x86_64`.:
+ifndef::openshift-origin[]
+.. Export the type of architecture for your server, such as `x86_64`:
```

Comment:
For OKD we don't set `ARCHITECTURE`, so `${OCP_RELEASE}-${ARCHITECTURE}` below needs to be just `${OCP_RELEASE}`

Reply:
Good catch, I'll update